web/static/js/reducers/questionnaire.js

```diff
@@ -108,19 +123,19 @@ const changeChoice = (state, action) => {
   }))
 }
 
-const autoComplete = (state, value) => {
+const autoComplete = (state, value, quiz: Questionnaire) => {
   let setted = false
 
   let smsValues = ''
   let ivrValues = ''
 
   state.forEach((step) => {
-    if (!setted) {
+    if ((step.type === 'multiple-choice') && !setted) {
```

Comment:
Watch this: again, for timing issues I limited autocompletes to regular Multiple Choice steps. I'm almost sure this should also be made to work with Language Selection steps but I had to cut corners.